pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
+import subprocess
+import logging
+import os
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+
+
+def test_integration_test(caplog):
+
+    caplog.set_level(logging.INFO)
+
+    ROOT_DIR = os.path.dirname(os.path.abspath("setup.py"))
+
+    for example_script in glob.glob(f"{ROOT_DIR}/examples/calculate_*.py"):
+
+        # run script and check that it doesn't crash
+        subprocess.check_output(f"python {example_script}", shell=True)
+
+        # check that the output file exists and it's not empty
+        outname = f"{ROOT_DIR}/pyirf_eventdisplay.fits.gz"
```

Comment:
use `os.path.join` for joining paths, or even better use `pathlib.Path`

Reply:
Also: here your globbing approach fails. other scripts probably won't have an output file name `pyirf_eventdisplay.fits.gz`